exercises/error-handling/common.go

```diff
@@ -2,6 +2,16 @@ package erratum
 
 import "io"
 
+// These are the support types and interface definitions used in the
+// implementation if your Use function. See the test suite file at
+// ./error_handling_test.go for information on the expected implementation.
+//
+// Because this are part of the package "erratum" if your solution file
```

Comment:
*Because this is part of the package "erratum", if your solution file* - change are to is, add comma.